massa-network-worker/src/peer_info_database.rs

```diff
@@ -1,5 +1,20 @@
 // Copyright (c) 2022 MASSA LABS <info@massa.net>
 
+//! Peer management (conection counts and storage)
+//!
+//! If you wish to provide initial peers, fillup the initial_peers.json file
```

Comment:
Yeah I'll open up a follow up and add it to the next episode checklist

Reply:
Ok I was thinking it would appear here https://massalabs.github.io/massa/massa_network_worker/ but it doesn't, because it's not public.

@yvan-sraka is there a way to display this doc comments in the generated doc ?